lib/has_safe_dates/core_ext.rb

```diff
@@ -54,28 +55,27 @@ def has_safe_dates(*args)
     end
   end
 
-  module MultiparameterAttributeExt
+  module DateTimeExt
     # Overrides #read_date when has_safe_dates is enabled for the current field the multiparameter.
     # Otherwise the original #read_date method is invoked.
-    def read_date
-      if ActiveRecord::Base.has_safe_fields_config[object.class.base_class] && ActiveRecord::Base.has_safe_fields_config[object.class.base_class][:fields].include?(name)
-        values.values_at(1,2,3).join("-")  # Convert multiparameter parts into a Date string, e.g. "2011-4-23", return it, and allow CoreExt methods handle the result.
-      else
-        super  # has_safe_dates is not enabled for the current field, so invoke the super method (original #read_date method).
-      end
-    end
-
-    # Overrides #read_time when has_safe_dates is enabled for the current field the multiparameter.
-    # Otherwise the original #read_date method is invoked.
-    def read_time
-      if ActiveRecord::Base.has_safe_fields_config[object.class.base_class] && ActiveRecord::Base.has_safe_fields_config[object.class.base_class][:fields].include?(name)
-        "#{ values.values_at(1,2,3).join("-")} #{ values.values_at(4,5).join(":") }"  # Convert multiparameter parts into a Time string, e.g. "2011-4-23 12:34", return it, and allow CoreExt methods handle the result.
+    def execute_callstack_for_multiparameter_attributes(callstack)
```

Comment:
I'm not 100% sure about this code. The class that was being modified in the original code was removed in rails 5, and its methods were refactored and put in other modules, so this section needed to be completely refactored. Specs pass though

Reply:
https://github.com/rails/rails/commit/631707a572fe14f3bbea2779cc97fcc581048d62#diff-06feeaad7a781bbbdb715d849ca76d51 <-- Here's the commit that removed the old code